HZ: 100

MAX_ROLL_ANGLE: 0.314159265359
MAX_PITCH_ANGLE: 0.314159265359
MAX_ROLL_VELOCITY: 0.5
MAX_PITCH_VELOCITY: 0.5